2d_classification/monai_101.ipynb

```diff
@@ -247,6 +259,7 @@
     "    optimizer=torch.optim.Adam(model.parameters(), lr=1e-5),\n",
     "    loss_function=torch.nn.CrossEntropyLoss(),\n",
     "    inferer=SimpleInferer(),\n",
+    "    train_handlers=StatsHandler(),\n",
```

Comment:
Do we need to add args for StatsHandler? I think you can refer to:
https://github.com/Project-MONAI/tutorials/blob/main/modules/engines/unet_training_dict.py#L169

Thanks.

Reply:
I think the default args may be sufficient:

Here is a demonstration
```
INFO:ignite.engine.engine.SupervisedTrainer:Engine run resuming from iteration 0, epoch 0 until 5 epochs
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 1/93 -- label: 3.0000 loss: 1.8036 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 2/93 -- label: 4.0000 loss: 1.7645 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 3/93 -- label: 0.0000 loss: 1.7261 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 4/93 -- label: 4.0000 loss: 1.7014 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 5/93 -- label: 0.0000 loss: 1.6741 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 6/93 -- label: 4.0000 loss: 1.6585 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 7/93 -- label: 1.0000 loss: 1.6306 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 8/93 -- label: 5.0000 loss: 1.5833 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 9/93 -- label: 5.0000 loss: 1.5724 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/5, Iter: 10/93 -- label: 2.0000 loss: 1.5359 
```